modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts

```diff
@@ -39,6 +39,7 @@ const get = (dom: DOMUtils) => {
         },
         inherit: false,
         preview: false,
+        // TODO: Look at deprecating this property?
```

Comment:
I think this might be something to add a Jira for?

Reply:
Have removed the `defaultBlock` property and any references to it as it is no longer useful. It doesn't appear to be documented anyway so didn't add a changelog entry but can add one if required